recipes/bison/all/conanfile.py

```diff
@@ -76,6 +77,7 @@ def _configure_autotools(self):
             "--enable-relocatable",
             "--disable-nls",
             "--datarootdir={}".format(os.path.join(self.package_folder, "bin", "share").replace("\\", "/")),
+            "--with-libreadline-prefix={}".format(tools.unix_path(self.deps_cpp_info["readline"].rootpath)),
```

Comment:
This will fail when using two profiles (M1), make something like:

```
    @property 
    def _user_info_build(self): 
        return getattr(self, "user_info_build", None) or self.deps_user_info 
```

and then

```
self._user_info_build["readline"].rootpath
```

I'm not 100% sure it will work, so please, test it before, it should work/raise even if you use two profiles that are the same.

Reply:
It doesn't for readline, it returns an empty dictionary. I've updated to automake.